src/backend/README.md

```diff
@@ -1,174 +1,135 @@
+
+Since Clio only extracts past validated ledger data from a group of trusted Rippled nodes, it can be safely assumed that these ledger data are correct without the need to validate with other  nodes in the XRP peer-to-peer network. Because of this, Clio is able to use a flattened data model to store the past validated ledger data, which allows for direct record lookup with much faster constant time operation time. 
+
+There are three parts of data in each XRP ledger version, they are [Ledger Header](https://xrpl.org/ledger-header.html), [Transaction Set](https://xrpl.org/transaction-formats.html) and [State Data](https://xrpl.org/ledger-object-types.html). Due to the structural differences of the different types of databases, Clio may choose to represent these data using a different schema for each unique database type. 
+
+**Keywords**
+*Sequence*: A unique incrementing identification number used to label the different ledger versions.
+*Hash*: The SHA512-half (calculate SHA512 and take the first 256 bits) hash of various ledger data like the entire ledger or specific ledger objects.  
+*Data*: The  [binary-encoded](https://xrpl.org/serialization.html) STObject  containing specific data (i.e. metadata, transaction data).
+*Metadata*: The data containing [detailed information](https://xrpl.org/transaction-metadata.html#transaction-metadata) of the outcome of a specific transaction, regardless of whether the transaction was successful.
+*Transaction data*:  The data containing the [full details](https://xrpl.org/transaction-common-fields.html) of a specific transaction. 
+*Object Index*: The pseudo-random unique identifier of a ledger object, created by hashing the data of the object.
+
+ 
+
+## Cassandra Implementation
+Cassandra is a distributed wide-column NoSQL database designed to handle large data throughput with high availability and no single point of failure. By leveraging Cassandra, Clio will be able to quickly and reliably scale up when needed simply by adding more Cassandra nodes to the Cassandra cluster configuration.
+
+In Cassandra, Clio will be creating 9 keyspaces to store the ledger data, they are `ledger_transactions`, `transactions`, `ledger_hashes`, `ledger_range`, `objects`, `ledgers`, `diff`, `account_tx`, and `successor`.  Their schemas and how they work are detailed below.
```

Comment:
```suggestion
In Cassandra, Clio will be creating 9 tables to store the ledger data, they are `ledger_transactions`, `transactions`, `ledger_hashes`, `ledger_range`, `objects`, `ledgers`, `diff`, `account_tx`, and `successor`.  Their schemas and how they work are detailed below.
```
These are called *tables* keyspace is more similar to a "database" in PostgreSQL.